//
// Created by חי כרמי on 14/06/2023.
//

#include "exception.h"

Exception::Exception(const string &problem) {
    this->problem = problem;
}
